52.md

```diff
@@ -90,9 +90,12 @@ The list of tags are as follows:
 * `end` (optional) exclusive end Unix timestamp in seconds. If omitted, the calendar event ends instantaneously.
 * `start_tzid` (optional) time zone of the start timestamp, as defined by the IANA Time Zone Database. e.g., `America/Costa_Rica`
 * `end_tzid` (optional) time zone of the end timestamp, as defined by the IANA Time Zone Database. e.g., `America/Costa_Rica`. If omitted and `start_tzid` is provided, the time zone of the end timestamp is the same as the start timestamp.
+* `about` (optional) description of the calendar event
```

Comment:
Why do we need `about` if we already have `content`?

Reply:
If so, I recommend that documenting the `about` tag as deprecated in NIP-24.

Reply:
Unfortunately some people are incapable of reading specs. Let's get rid of `content` and just use `about` everywhere?

Reply:
I disagree. Let's just update Flockstr to use content. I'm building a Comingle, a mobile events app. All the calendar events I've seen have content populated already. It's a trivial change and won't break existing clients. @zmeyer44  what do you think?

Reply:
I like the idea of content being used for a more expressive and detailed account of the event (possibly in Markdown) while about can serve as a short description, perhaps something that would be used in a preview card.

Reply:
Not opposed. [Long-form notes](https://github.com/nostr-protocol/nips/blob/master/23.md) do that. It uses a summary tag for a short description, and content for the full Markdown.

Reply:
I'm for changing 'about' to 'summary' if that would make things more clear and consistent

Reply:
I've released Comingle to Apple TestFlight, which supports `summary`.